### PR TITLE
feat(readme): Readme updates

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
Revamped readme:
- Added quickstart section
- Added community section
- Added Help section
- General cleanup

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+Check out a new branch, make modifications, run linters and tests, and push the branch to your fork:
+
+```sh
+$ git checkout -b <YOUR_BRANCH_NAME>
+# edit files
+$ make test
+$ git add
+$ git commit
+$ git push <YOUR_FORK> <YOUR_BRANCH_NAME>
```
Comment (dehaansa) [inline CONTRIBUTING.md:31]: Does this work? I've always typed `--set-upstream <fork>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Commands to run before submitting PR
+
+Our CI runs the following checks on each PR. You can run the following local commands to ensure your code is ready for PR:
+
+- Build (`make collector`)
+- CI Checks (`make ci-checks`)
+
+
+### How to Receive Comments
```
Comment (dehaansa) [inline CONTRIBUTING.md:44]: Feels like this section should be "How to Receive Feedback" or perhaps the first "if the PR is not ready for review" just doesn't quite feel like it fits.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+* If the PR is not ready for review, mark it as [`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* If you're stuck, tag a maintainer and ask a question. We're here to help each other.
+* Make sure CI passes.
+
+
+### How to Get PRs Merged
```
Comment (dehaansa) [inline CONTRIBUTING.md:51]: Could use a something at the end of the section like "Tag a maintainer to request a merge once this is all complete"

File: README.md [hunk 4]
```diff
@@ -1,60 +1,93 @@
...
+[![Action Test Status](https://github.com/observIQ/observiq-otel-collector/workflows/Tests/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/observIQ/observiq-otel-collector)](https://goreportcard.com/report/github.com/observIQ/observiq-otel-collector)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Gosec](https://github.com/observIQ/observiq-otel-collector/actions/workflows/gosec.yml/badge.svg)](https://github.com/observIQ/observiq-otel-collector/actions/workflows/gosec.yml)
+
+</center>
+
+## About
+
 The observIQ OpenTelemetry Collector is observIQ's distribution of the OpenTelemetry Collector.
```
Comment (dehaansa) [inline README.md:15]: Links/references back to main Otel Collector/Otel & observIQ would look good here to me.

Comment (cpheps) [reply README.md:15]: Yeah Mike is going to give me a better version of this down the road. He said this is fine to ship for now.

**Review by dehaansa — APPROVED** [review]

**Pull request merged**